apps/teams-test-app/src/components/PagesConfigAPIs.tsx

```diff
@@ -85,10 +85,10 @@ const SetValidityState = (): React.ReactElement =>
     },
   });
 
-const RegisterOnRemoveHandler = (): React.ReactElement =>
+const RegisterOnRemoveHandlerSuccess = (): React.ReactElement =>
```

Comment:
We're going to need to keep the `RegisterOnRemoveHandler` element while we add the others or the tests looking for it will fail. :)